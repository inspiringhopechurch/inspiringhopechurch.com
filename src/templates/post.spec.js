import React from "react";
import { shallow } from "enzyme";
import Post from "./post";
import data from "../fixtures/data";

describe("Post", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Post data={data.data} />);
    expect(wrapper).toMatchSnapshot();
  });
});
