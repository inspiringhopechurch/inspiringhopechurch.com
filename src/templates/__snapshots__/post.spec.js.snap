// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post should render without crashing 1`] = `
<Fragment>
  <SEO
    article={true}
    banner={null}
    desc="THE OFFICIAL NEWSLETTER OF THE MANGRUM FAMILY MINISTRY"
    pathname="our-church-planting-journey-volume-6"
    title="Our Church Planting Journey: Volume 6"
  />
  <section
    className="post-page hero is-halfheight"
  >
    <div
      className="hero-body"
      style={
        Object {
          "background": "url('undefined') center top",
          "backgroundSize": "cover",
        }
      }
    >
      <div
        className="post-title container"
      >
        <h1
          className="title"
        >
          Our Church Planting Journey: Volume 6
        </h1>
        <h3
          className="subtitle post-creation-info"
        >
          Inspiring Hope Church
           
          ﹒ 2
           min read
        </h3>
      </div>
    </div>
  </section>
  <section
    className="section post-content"
  >
    <div
      className="columns is-centered"
    >
      <div
        className="column content is-two-thirds"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>The summer is off to a great start! Even though it is definitely different this year, we have taken advantage of our state opening up some. We've made some more connections by meeting more neighbors, talking and working with local pastors on the same mission, and really trying to get involved in the community. Our parks have opened back up so we've visited the playgrounds a couple of times. We are planning on doing some community outreaches this next month by providing our local community workers with gift bags and treats to say thank you and handing out snow cones and/or ice cream in our neighborhood and surrounding communities. We want to let people know that Inspiring Hope Church is here and <em>we want them to be a part of what God is doing</em> through this church in <em>their</em> city.</p><p>We are also planning a neighborhood barbecue in our backyard. We are trying to keep crowds smaller so we are starting with smaller get-togethers and are hoping as things get better we can add some bigger events. We have had several couples/families to our home over the last month to build relationships and get to know Hamilton and Ohio. It's been so fun and such a blessing! Our Monday night Inspire Group is going strong. This has been an amazing time of fellowship, planning, discipleship, studying, and prayer for our team. We can't wait to see what the next month will bring!</p><h2 id=\\"pleasepray\\">PLEASE PRAY:</h2>
<ul>
<li>FOR OUR UPCOMING NEIGHBORHOOD BARBECUE</li>
<li>FOR OUR INSPIRE GROUP AS WE CONTINUE TO ENCOURAGE AND EQUIP THOSE THAT GATHER.</li>
<li>FOR OUR FAMILY AS WE HAVE DENTAL AND HEALTH PROCEDURES<br />
COMING UP.</li>
<li>FOR OUR TEAM AS GOD CONTINUES TO ADD MEMBERS.</li>
</ul>
<h2 id=\\"a-god-story-\\">A GOD STORY...</h2><p>As I'm sitting here and typing this newsletter, I'm reflecting on not only what God has done over this past month but what He has and continues to do over the entire duration of this journey. I am still so amazed by His goodness which I know is right where He wants us. He continues to build His church and we continue to be in awe and feel extremely blessed to get to be a part of it. He has opened so many doors and as we walk through them we have met different people, received so much love and kindness, and received wise counsel along the way. Our \\"God Story\\" this month is being reminded He is always and completely present. </p><figure class=\\"kg-card kg-bookmark-card\\"><a class=\\"kg-bookmark-container\\" href=\\"https://my.bible.com/bible/116/DEU.31.8\\"><div class=\\"kg-bookmark-content\\"><div class=\\"kg-bookmark-title\\">Deuteronomy 31:8 Do not be afraid or discouraged, for the LORD will personally go ahead of you. He will be with you; he will neither fail you nor abandon you.” | New Living Translation (NLT) | Download The Bible App Now</div><div class=\\"kg-bookmark-description\\">Do not be afraid or discouraged, for the LORD will personally go ahead of you. He will be with you; he will neither fail you nor abandon you.”</div><div class=\\"kg-bookmark-metadata\\"><span class=\\"kg-bookmark-publisher\\">Bible</span></div></div><div class=\\"kg-bookmark-thumbnail\\"><img src=\\"https://imageproxy.youversionapi.com/1280x1280/https://s3.amazonaws.com/static-youversionapi-com/images/base/3084/1280x1280.jpg\\" /></div></a></figure><h2 id=\\"many-thanks-\\">MANY THANKS...</h2><p>Thank you so much from the bottom of our hearts for choosing to partner with our family in planting Inspiring Hope Church in Ohio. We know that this endeavor would not be possible without the love and support of each one of you. Thank you for allowing God to use you to help us and being willing to give to this ministry. You are all such a blessing to us! Thank you. <strong>Thank you</strong>. <em>Thank you</em>!</p>",
          }
        }
      />
    </div>
  </section>
</Fragment>
`;
